### ISSUE TITLE
Specified munge key ignored

### ISSUE THREAD

**Issue by sjpb** [stackhpc/ansible-slurm-appliance#60]
Problem: `secrets_openhpc_mungekey` in `<env>/inventory/group_vars/all/secrets.yml`, is used to set `slurm_munge_key` in `/common/inventory/group_vars/all/openhpc.yml`. However current `stackhpc.openhpc` role takes `openhpc_munge_key_path`, not `slurm_munge_key.

Consequence: Specified munge key from secrets is not used.

Impact: None immediately as long as ansible is run over the cluster, as this distributes the control node's munge key anyway. The key in packer-built images is wrong, but that isn't critical as the ansible is run over all hosts to reconfigure slurm (although you see changes occur which you would not expect). However it will cause issues for joining nodes to the cluster without using ansible, e.g. autoscaling.

**Comment by sjpb** [comment]
Role PR is https://github.com/stackhpc/ansible-role-openhpc/pull/99.

If accepted, fix here is to add:

```
environments/common/inventory/group_vars/all/openhpc.yml:

openhpc_munge_key: "{{ secrets_openhpc_mungekey }}"
```

Nothing else needed.

**Comment by sjpb** [comment]
Fixed in #77
